2025-09-30T04:48:24 {"cpu": 12.82568359375, "cpu_total": 8, "cpu_usage": 1.60321044921875, "memory": 6846.0, "memory_total": 11944.0, "memory_usage": 0.5731748158070998, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759207704.173724, "uuid": "21513812c079457db1311660445b86f0"}
2025-09-30T04:48:25 {"block_run_id": 1130, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Execute PipelineRun 394, BlockRun 1130: pipeline ny_taxi_pipeline block ingest_taxi_trips", "timestamp": 1759207705.182697, "uuid": "58d326a8ed9e43a7ae425b9f21744823"}
2025-09-30T04:48:25 {"cpu": 12.7939453125, "cpu_total": 8, "cpu_usage": 1.5992431640625, "memory": 6917.0, "memory_total": 11944.0, "memory_usage": 0.5791192230408574, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759207705.589958, "uuid": "ff936f9181f84073bf93239f76541751"}
2025-09-30T04:48:34 {"cpu": 12.751953125, "cpu_total": 8, "cpu_usage": 1.593994140625, "memory": 7019.0, "memory_total": 11944.0, "memory_usage": 0.5876590756865372, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "2b86a123826c", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759207714.128851, "uuid": "8c7897624d7a4fa6a4d14dd5a99e7547"}
2025-09-30T12:35:04 {"cpu": 0.20654296875, "cpu_total": 8, "cpu_usage": 0.02581787109375, "memory": 1251.0, "memory_total": 11944.0, "memory_usage": 0.10473878097789685, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235704.958725, "uuid": "3cd3d0580ede40ff8a6205846dc4c24e"}
2025-09-30T12:35:06 {"block_run_id": 1130, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Execute PipelineRun 394, BlockRun 1130: pipeline ny_taxi_pipeline block ingest_taxi_trips", "timestamp": 1759235706.648977, "uuid": "2eeb55db540942c6ad09f7afa06d2934"}
2025-09-30T12:35:15 {"cpu": 0.2744140625, "cpu_total": 8, "cpu_usage": 0.0343017578125, "memory": 1481.0, "memory_total": 11944.0, "memory_usage": 0.12399531145344943, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235715.487916, "uuid": "1dd7079146574dc29be5e25c493fe9f8"}
2025-09-30T12:35:25 {"cpu": 0.34130859375, "cpu_total": 8, "cpu_usage": 0.04266357421875, "memory": 1573.0, "memory_total": 11944.0, "memory_usage": 0.13169792364367047, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235725.913235, "uuid": "edcedc0ed0394a51a60cf7dfd1facc86"}
2025-09-30T12:35:37 {"cpu": 0.40673828125, "cpu_total": 8, "cpu_usage": 0.05084228515625, "memory": 1664.0, "memory_total": 11944.0, "memory_usage": 0.13931681178834562, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235737.587601, "uuid": "db98b3bc43df496f9aaf973b9dc1bd2b"}
2025-09-30T12:35:45 {"cpu": 0.431640625, "cpu_total": 8, "cpu_usage": 0.053955078125, "memory": 1729.0, "memory_total": 11944.0, "memory_usage": 0.14475887474882787, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235745.438157, "uuid": "7957bba062624c839483dddcb263f631"}
2025-09-30T12:35:55 {"cpu": 0.48095703125, "cpu_total": 8, "cpu_usage": 0.06011962890625, "memory": 1795.0, "memory_total": 11944.0, "memory_usage": 0.150284661754856, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235755.388022, "uuid": "573b971ca52344f1abfc032ea46f9cd4"}
2025-09-30T12:36:05 {"cpu": 0.4921875, "cpu_total": 8, "cpu_usage": 0.0615234375, "memory": 1850.0, "memory_total": 11944.0, "memory_usage": 0.15488948425987945, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235765.49003, "uuid": "a3325ebfd37c42759b79555a72ee1479"}
2025-09-30T12:36:16 {"cpu": 0.59765625, "cpu_total": 8, "cpu_usage": 0.07470703125, "memory": 3912.0, "memory_total": 11944.0, "memory_usage": 0.3275284661754856, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235776.824481, "uuid": "5a5d194211e8428798bbd98661422ca0"}
2025-09-30T12:36:26 {"cpu": 0.671875, "cpu_total": 8, "cpu_usage": 0.083984375, "memory": 6856.0, "memory_total": 11944.0, "memory_usage": 0.5740120562625586, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235786.656271, "uuid": "3d932598ab6a4532a53fbe7357c0d9de"}
2025-09-30T12:36:35 {"cpu": 0.6845703125, "cpu_total": 8, "cpu_usage": 0.0855712890625, "memory": 9175.0, "memory_total": 11944.0, "memory_usage": 0.7681681178834562, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235795.331641, "uuid": "469305ad89b44adc82f96513da42325d"}
2025-09-30T12:36:45 {"cpu": 0.71533203125, "cpu_total": 8, "cpu_usage": 0.08941650390625, "memory": 10122.0, "memory_total": 11944.0, "memory_usage": 0.8474547890154053, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235805.980224, "uuid": "7c0d0c20495e44868c48d2bc2b69b8ae"}
2025-09-30T12:36:55 {"cpu": 0.76708984375, "cpu_total": 8, "cpu_usage": 0.09588623046875, "memory": 10076.0, "memory_total": 11944.0, "memory_usage": 0.8436034829202947, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235815.698813, "uuid": "e97227a798934679b471624034c8c5b9"}
2025-09-30T12:37:04 {"cpu": 0.8076171875, "cpu_total": 8, "cpu_usage": 0.1009521484375, "memory": 10488.0, "memory_total": 11944.0, "memory_usage": 0.8780977896851976, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235824.974538, "uuid": "2688c653c1cb4d07b1a66f9cfa7c5bf0"}
2025-09-30T12:37:16 {"cpu": 0.94970703125, "cpu_total": 8, "cpu_usage": 0.11871337890625, "memory": 10178.0, "memory_total": 11944.0, "memory_usage": 0.8521433355659745, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235836.324312, "uuid": "664dbbac58244c31951c61a871ce7a8a"}
2025-09-30T12:37:25 {"cpu": 1.4228515625, "cpu_total": 8, "cpu_usage": 0.1778564453125, "memory": 7768.0, "memory_total": 11944.0, "memory_usage": 0.6503683858004019, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235845.509895, "uuid": "64d3dbfe97c74c1db476ac36cc377fe6"}
2025-09-30T12:37:25 {"pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "block_run_id": 1130, "block_uuid": "ingest_taxi_trips", "hostname": "3d7c559945b8", "level": "INFO", "message": "Process 71 is dead for job block_run_1130", "timestamp": 1759235845.695065, "uuid": "7c2b72e250244d85a6bee92aef3ef576"}
2025-09-30T12:37:26 {"block_run_id": 1130, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Execute PipelineRun 394, BlockRun 1130: pipeline ny_taxi_pipeline block ingest_taxi_trips", "timestamp": 1759235846.176372, "uuid": "30247175766a451f9af4582cde9797f1"}
2025-09-30T12:37:30 {"block_run_id": 1130, "block_uuid": "ingest_taxi_trips", "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "EXCEPTION", "message": "BlockRun 1130 (block_uuid: ingest_taxi_trips) failed.", "timestamp": 1759235850.018433, "uuid": "728e29ffc0184fe2986c8acedcc366d3", "error": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 647, in execute\n    result = __execute_with_retry()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 54, in retry_func\n    raise e\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py\", line 38, in retry_func\n    return func(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 621, in __execute_with_retry\n    return self._execute(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 1154, in _execute\n    result = self.block.execute_sync(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1691, in execute_sync\n    return __execute()\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1668, in __execute\n    raise err\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1570, in __execute\n    output = self.execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 1910, in execute_block\n    outputs = self._execute_block(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 2073, in _execute_block\n    outputs = self.execute_block_function(\n  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py\", line 2169, in execute_block_function\n    output = block_function_updated(*input_vars, **global_vars)\n  File \"<string>\", line 18, in load_data_from_file\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py\", line 503, in read_parquet\n    return impl.read(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py\", line 244, in read\n    path_or_handle, handles, kwargs[\"filesystem\"] = _get_path_or_handle(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py\", line 102, in _get_path_or_handle\n    handles = get_handle(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/common.py\", line 713, in get_handle\n    ioargs = _get_filepath_or_buffer(\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/common.py\", line 363, in _get_filepath_or_buffer\n    with urlopen(req_info) as req:\n  File \"/usr/local/lib/python3.10/site-packages/pandas/io/common.py\", line 265, in urlopen\n    return urllib.request.urlopen(*args, **kwargs)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 216, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 525, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 634, in http_response\n    response = self.parent.error(\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 563, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 496, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.10/urllib/request.py\", line 643, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 403: Forbidden\n", "error_stack": ["  File \"/usr/local/bin/mage\", line 8, in <module>\n    sys.exit(app())\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 311, in __call__\n    return get_command(self)(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1442, in __call__\n    return self.main(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 778, in main\n    return _main(\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/core.py\", line 216, in _main\n    rv = self.invoke(ctx)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1830, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 1226, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n", "  File \"/usr/local/lib/python3.10/site-packages/click/core.py\", line 794, in invoke\n    return callback(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/typer/main.py\", line 683, in wrapper\n    return callback(**use_params)  # type: ignore\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/cli/main.py\", line 147, in start\n    start_server(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/server.py\", line 809, in start_server\n    scheduler_manager.start_scheduler()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 108, in start_scheduler\n    proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/server/scheduler_manager.py\", line 60, in run_scheduler\n    LoopTimeTrigger().start()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/loop_time_trigger.py\", line 14, in start\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/triggers/time_trigger.py\", line 11, in run\n    schedule_all()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1819, in schedule_all\n    PipelineScheduler(r).schedule()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 159, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 334, in schedule\n    self.__schedule_blocks(block_runs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 641, in __schedule_blocks\n    job_manager.add_job(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/job_manager.py\", line 21, in add_job\n    self.queue.enqueue(job_id, target, *args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 122, in enqueue\n    self.start_worker_pool()\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 199, in start_worker_pool\n    self.worker_pool_proc.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 108, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 353, in poll_job_and_execute\n    worker.start()\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 121, in start\n    self._popen = self._Popen(self)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 224, in _Popen\n    return _default_context.get_context().Process._Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/context.py\", line 281, in _Popen\n    return Popen(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 19, in __init__\n    self._launch(process_obj)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/popen_fork.py\", line 71, in _launch\n    code = process_obj._bootstrap(parent_sentinel=child_r)\n", "  File \"/usr/local/lib/python3.10/multiprocessing/process.py\", line 314, in _bootstrap\n    self.run()\n", "  File \"/usr/local/lib/python3.10/site-packages/newrelic/api/background_task.py\", line 117, in wrapper\n    return wrapped(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/queue/process_queue.py\", line 317, in run\n    start_session_and_run(args[1], *args[2], **args[3])\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/process.py\", line 15, in start_session_and_run\n    results = target(*args)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 1311, in run_block\n    return ExecutorFactory.get_block_executor(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py\", line 682, in execute\n    on_failure(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/db/__init__.py\", line 159, in func_with_rollback\n    return func(*args, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/orchestration/pipeline_scheduler_original.py\", line 465, in on_block_failure\n    self.logger.exception(\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 30, in exception\n    self.__send_message('exception', message, **kwargs)\n", "  File \"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/logging/logger.py\", line 65, in __send_message\n    data['error_stack'] = traceback.format_stack()\n"], "error_stacktrace": "HTTP Error 403: Forbidden"}
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 647, in execute
    result = __execute_with_retry()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 54, in retry_func
    raise e
  File "/usr/local/lib/python3.10/site-packages/mage_ai/shared/retry.py", line 38, in retry_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 621, in __execute_with_retry
    return self._execute(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/executors/block_executor.py", line 1154, in _execute
    result = self.block.execute_sync(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1691, in execute_sync
    return __execute()
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1668, in __execute
    raise err
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1570, in __execute
    output = self.execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 1910, in execute_block
    outputs = self._execute_block(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 2073, in _execute_block
    outputs = self.execute_block_function(
  File "/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/block/__init__.py", line 2169, in execute_block_function
    output = block_function_updated(*input_vars, **global_vars)
  File "<string>", line 18, in load_data_from_file
  File "/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py", line 503, in read_parquet
    return impl.read(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py", line 244, in read
    path_or_handle, handles, kwargs["filesystem"] = _get_path_or_handle(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/parquet.py", line 102, in _get_path_or_handle
    handles = get_handle(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/common.py", line 713, in get_handle
    ioargs = _get_filepath_or_buffer(
  File "/usr/local/lib/python3.10/site-packages/pandas/io/common.py", line 363, in _get_filepath_or_buffer
    with urlopen(req_info) as req:
  File "/usr/local/lib/python3.10/site-packages/pandas/io/common.py", line 265, in urlopen
    return urllib.request.urlopen(*args, **kwargs)
  File "/usr/local/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/local/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/local/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/local/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/local/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
2025-09-30T12:37:39 {"cpu": 1.49169921875, "cpu_total": 8, "cpu_usage": 0.18646240234375, "memory": 10339.0, "memory_total": 11944.0, "memory_usage": 0.8656229068988613, "pipeline_run_id": 394, "pipeline_schedule_id": 5, "pipeline_uuid": "ny_taxi_pipeline", "hostname": "3d7c559945b8", "level": "INFO", "message": "Pipeline ny_taxi_pipeline for run 394 in schedule 5 is alive.", "timestamp": 1759235859.186926, "uuid": "ef853f64656c40b792148f3d37d33162"}
